kingdom-of-ocelader/TrestlebridgeFarms
# oops

# Description

Added plowed field class that allows you to add a plowed field facility

Applies to issue # (12)

## Type of change

Please delete options that are not relevant.




# Testing Instructions
Run application and make sure you can add a new plowed field and shows a message that you have added a new field.



# Checklist:
